app/views/spotlight/searches/_form.html.erb

```diff
@@ -30,6 +30,10 @@
       <div role="tabpanel" class="tab-pane active" id="search-description">
         <%= f.text_field :title, control_col: "col-sm-5" %>
         <%= f.text_area :long_description, rows: 5 %>
+        <%= f.form_group :search_box, label: { text: t(:'.search_box.label'), class: nil }, help: nil do %>
+          <%= f.check_box :search_box, label: "" %>
+          <p class="help-block"><%= t(:'.search_box.help_block') %></p>
```

Comment:
Is this functionally different than setting ` t(:'.search_box.help_block')` as the `help` key for the `form_group` itself?  I think `bootstrap_form` may be able to handle this for us.

Reply:
👍  good point. I've added the key to `form_group` and rebased.